mod1/src/file-4.xoo

```diff
@@ -0,0 +1,7 @@
+// dummy line 1
+// dummy line 2
+// dummy line 3
+// dummy line 4
+// dummy line 5
+// dummy line 6
+// violate HasTag rule: xoo on line 7
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Has Tag ([xoo:HasTag](http://127.0.0.1:17058/coding_rules#rule_key=xoo%3AHasTag))

[See it in SonarQube](http://127.0.0.1:17058/project/issues?id=sample&branch=feature4&issues=AWC2Duyx50_BW5zAiCAa&open=AWC2Duyx50_BW5zAiCAa)

[ISSUE_KEY]: AWC2Duyx50_BW5zAiCAa